src/components/Reports/Reports.js

```diff
@@ -32,20 +33,18 @@ class Reports extends React.Component {
 
   async componentDidMount() {
     const domains = await this.datahubClient.getAccountDomains();
-    if (domains.length) {
-      const domainSelected = domains[0];
-      const pages = [];
-      const pageSelected = pages.length ? pages[0] : null;
-      let dateFrom = new Date();
-      dateFrom.setDate(dateFrom.getDate() - parseInt(this.state.periodSelectedDays));
-      this.setState({
-        domains: domains,
-        domainSelected: domainSelected,
-        pages: pages,
-        pageSelected: pageSelected,
-        dateFrom: dateFrom,
-      });
-    }
+    const domainSelected = domains.length ? domains[0] : null;
+    const pages = [];
```

Comment:
no need and if here? if doesn't has domains not pages too..

Reply:
I am thinking about removing all pages related code by the moment...

Reply:
I think that this solution is ugly, but safe regarding to that, so, by the moment I will keep it as it is.

Thanks @fcoronelmakingsense !